deployments/gpu-operator/templates/clusterpolicy.yaml

```diff
@@ -12,6 +12,10 @@ metadata:
     "helm.sh/resource-policy": keep
   {{- end }}
 spec:
+  hostPaths:
+    root: {{ .Values.hostPaths.root | default "/" }}
+    driverInstallDir: {{ .Values.hostPaths.driverInstallDir | default "/run/nvidia/driver" }}
+    devPath: {{ .Values.hostPaths.devPath | default "" }}
```

Comment:
Should the default be `/run/nvidia/driver/dev` to be explicit?

Reply:
I have removed devPath from the ClusterPolicy API now. Consequently, this code has been removed.